bittensor/core/settings.py

```diff
@@ -53,30 +53,65 @@ def turn_console_on():
 turn_console_off()
 
 bt_console = __console__
+bt_err_console = Console(stderr=True)
+bt_verbose_console = Console(quiet=True)
+
+
+def print_console(message: str, colour: str, title: str, console: Console):
+    console.print(
+        f"[bold {colour}][{title}]:[/bold {colour}] [{colour}]{message}[/{colour}]\n"
+    )
+
+
+def print_verbose(message: str, status=None):
+    """Print verbose messages while temporarily pausing the status spinner."""
+    if status:
+        status.stop()
+        print_console(message, "green", "Verbose", bt_verbose_console)
+        status.start()
+    else:
+        print_console(message, "green", "Verbose", bt_verbose_console)
+
+
+def print_error(message: str, status=None):
+    """Print error messages while temporarily pausing the status spinner."""
+    if status:
+        status.stop()
+        print_console(message, "red", "Error", bt_err_console)
+        status.start()
+    else:
+        print_console(message, "red", "Error", bt_err_console)
```

Comment:
Does this make sense in an SDK?

Reply:
At this step/part, I want to make everything work and not break the async logic that I took from the btcli and changed a bit. 
Now any `console` based functions works and performs the same functions as btlogging. I can do this refactoring either in the next step or during the creation of the common package. I would answer both remaining comments this way.

Reply:
After some thought, I applied all the comments.

NOTE: Do not update the branch until the `testnet` is updated from `devnet`

Reply:
Merge after https://github.com/opentensor/bittensor/pull/2377 is merged